server/remote/gitea/helper.go

```diff
@@ -107,14 +111,12 @@ func buildFromPush(hook *pushHook) *model.Build {
 func getChangedFilesFromPushHook(hook *pushHook) []string {
 	files := make([]string, 0)
 
-	if len(hook.Commits) == 0 {
-		return files
+	for _, c := range hook.Commits {
+		files = append(files, c.Added...)
+		files = append(files, c.Removed...)
+		files = append(files, c.Modified...)
 	}
```

Comment:
Good catch. I would like to extract it and check with other SCMs as well.

Reply:
well I'm not sure if we can extract it because hook format is different for each forge (most similarity's are shared between github and gitea ...)

Reply:
Sorry, meant extract into a separate PR not code-wise.

Reply:
@anbraten done